pom.xml

```
@@ -28,7 +28,7 @@
     </scm>
     <properties>
         <revision>2.3.x</revision>
```

As explained, left on purpose.

---

https://github.com/jenkinsci/jaxb-plugin/blob/1cd47382d83631f6bef4a02a675622519cdcda05/pom.xml#L39-L53 does not smell right. Why are these versions different? Is there a BOM we could import? If not, and these three versions are all actually floating, I would recommend
```suggestion
        <revision>2.3</revision>
```

and then use Enforcer to verify that the versions of these three components all start with that prefix.

---

they are different although looks like core does have a `2.3.0.1` available

---

Or is there one component (like `jaxb-impl`) which can be the sole import, where the others are transitive dependencies in suitably new versions?

Alternately, if there is not really a well-defined “version of JAXB”, then maybe the plain `${changelist}` is more appropriate.

---

> Is there a BOM we could import?

=> #19